src/main/java/GPT/EventCommand.java

```diff
@@ -0,0 +1,62 @@
+package GPT;
+import java.time.LocalDateTime;
+
+/**
+ * Represents a command to add a new event task in the GPT application.
+ * This command parses the user input to extract the task description, start time, and end time,
+ * and adds the task to the task list if the input is valid.
+ */
+public class EventCommand extends Command {
+    private String description;
+    private String from;
+    private String to;
+
+    /**
+     * Constructs an EventCommand by parsing the user input to extract the task description,
+     * start time, and end time. If the input format is invalid, the description, start time,
+     * and end time will be set to null.
+     *
+     * @param input The user input string containing the command to add an event task.
+     */
+    public EventCommand(String input) {
+        String[] parts = input.substring(5).trim().split("/from|/to");
+        if (parts.length < 3 || parts[0].trim().isEmpty() || parts[1].trim().isEmpty() || parts[2].trim().isEmpty()) {
```

Comment:
I think this code of line can be separated into 2 lines!
I think this phenomenon (code of line should be split as the number of characters exceed 110)  can be found in other files as well for you to consider whether to rectify them!

Reply:
Thank you for pointing this out! I have gone through the codebase and resolved all similar issues where the lines exceeded the character limit. The lines have been split accordingly for better readability and to follow the coding standards.